lib/engine.js

```diff
@@ -8,6 +8,8 @@ var error = debug('nemo:engine:error');
 var uuidv4 = require('uuid/v4');
 var masterID = uuidv4();
 var Storage = require('./storage');
+const EventEmitter = require('events');
+class Emitter extends EventEmitter {}
```

Comment:
Something I've been thinking about in general for a lot of the stuff that's being passed around:
Maybe we can just make a singleton event emitter in an `event.js` module which exposes `trigger` and `on/subscribe` so the whole app can import it without needing to think hard about getting on the config object we're passing everywhere.